content/docs/5.1/deploy/operate/upgrade.md

```diff
@@ -17,15 +17,14 @@ This section introduces the preparation works needed before upgrading your TiKV
 
 Before upgrading your TiKV cluster, you first need to upgrade TiUP.
 
-1. Upgrade the TiUP version. It is recommended that the TiUP version is `1.4.0` or later.
+1. Upgrade the TiUP version. It is recommended that the TiUP version be `1.4.0` or later.
 
     ```bash
     tiup update --self
     tiup --version
     ```
 
-2. Upgrade the TiUP Cluster version. It is recommended that the TiUP Cluster version is `1.4.0` or later.
-
+2. Upgrade the TiUP Cluster version. It is recommended that the TiUP Cluster version be `1.4.0` or later.
```

Comment:
This line should be consistent with Line 20 in style?

Reply:
Agree